src/components/Book.js

```diff
@@ -13,26 +16,51 @@ const SingleBook = (props) => {
 
   return (
     <li>
-      <div className="book">
-        <div className="book-content">
-          <div className="book-info">
-            <h2 className="book-title">{title}</h2>
-            <h2 className="book-author">{author}</h2>
+    <div className="book">
+      <div className="book-content">
+        <div className="book-info">
+          <h4 className="book-category">{Category}</h4>
+          <h2 className="book-title">{title}</h2>
+          <h2 className="book-author">{author}</h2>
+          <div className="action-buttons">
+            <button type="button" className="btn-outline">Comments</button>
+            <div className="vertical-divider" />
+            <button type="button" className="btn-outline" onClick={handleRemoveBook}>Remove</button>
+            <div className="vertical-divider" />
+            <button type="button" className="btn-outline">Edit</button>
+          </div>
+        </div>
+        <div className="progress-container">
+          <div className="circular-progress-container">
+            <div className="circular-progress" />
```

Comment:
- [ ] I see that you made significant efforts to fix the issues raised by the previous reviewer but you created other issues. 
- The navbar should have a flexible width like it was before the first reviewer. 
- The Book component should have equal padding on both left and right. Currently, you have the correct padding on the left but no padding on the right.
- The add book form should have a flexible width that spans the with of the screen. 

Honestly, I would love to approve your work, but with the issues shown below, I cannot do so. Kindly ensure you take out time to fix all these things and make your work look the same as the Zeplin design. 


![image](https://user-images.githubusercontent.com/84156714/233808990-fc937477-a147-4de7-872d-b15f98fdd49e.png)

Reply:
@bobb-Rob Hello, the submission deadline is nearing. I cannot afford to wait another 40 minutes for the next review. these issues are minor with respect to time. PLease help me out

Reply:
@bobb-Rob If I wait for another review, the deadline will pass and I will be unable to submit

Then I will have to repeat the entire module because of margin spaces.. I've really struggled with this project, if not I won't be submitting so late. I have done so much to implement everything and get it working. It is unfair to repeat the module because of margin right and width. PLEASE, I AM BEGGING YOU, consider me